tests/drivers/i2c/i2c_api/testcase.yaml

```diff
@@ -1,7 +1,6 @@
 tests:
-  test_i2c_arc:
-    platform_whitelist: quark_se_c1000_ss_devboard
-    tags: drivers
-  test_i2c_x86:
-    platform_whitelist: quark_se_c1000_devboard quark_d2000_crb
-    tags: drivers
+  test_i2c:
+    depends_on: i2c
+    tags: drivers i2c
+    harness: sensor
+    platform_whitelist: quark_se_c1000_devboard quark_d2000_crb quark_se_c1000_ss_devboard # FIXME
```

Comment:
maybe amend the FIXME to state what needs to be changed and why the platform_whitelist currently remains.